docker-compose.yml

```diff
@@ -88,30 +91,30 @@ services:
   extractors-heartbeat:
     image: "clowder/clowder2-heartbeat"
     build:
-      context: ./backend
+      context: backend
       dockerfile: heartbeat.Dockerfile
     networks:
       - clowder2
     restart: unless-stopped
     environment:
-      - MONGODB_URL=mongodb://mongo:27017
-      - RABBITMQ_HOST=rabbitmq
-      - RABBITMQ_USER=${RABBITMQ_USER:-guest}
-      - RABBITMQ_PASS=${RABBITMQ_PASS:-guest}
+      MONGODB_URL: mongodb://mongo:27017
+      RABBITMQ_HOST: ${RABBITMQ_HOST:-rabbitmq}
+      RABBITMQ_USER: ${RABBITMQ_USER:-guest}
+      RABBITMQ_PASS: ${RABBITMQ_PASS:-guest}
     depends_on:
       - mongo
       - rabbitmq
 
   extractors-messages:
     image: "clowder/clowder2-messages"
     build:
-      context: ./backend
+      context: backend
       dockerfile: messages.Dockerfile
     environment:
-      - MONGODB_URL=mongodb://mongo:27017
-      - RABBITMQ_HOST=rabbitmq
-      - RABBITMQ_USER=${RABBITMQ_USER:-guest}
-      - RABBITMQ_PASS=${RABBITMQ_PASS:-guest}
+      MONGODB_URL: mongodb://mongo:27017
```

Comment:
Just a query..... I noticed there was a difference in pattern there env were written. I guess it didn't make any changes to the workflow apart from aesthetics, right?

Reply:
one is a map and the other a list in yml. I was trying to see it would make a difference with the interpolation, but I think both seem to work fine.